### DISCUSSION TITLE
I have a problem with HasFileAttachments aws S3

### DISCUSSION THREAD

**Discussion by elmudometal** [filamentphp/filament#11008] [Help]
Filament v2.17.55
Laravel v9.19
PHP: 8.1

in the file vendor/filament/forms/src/Components/Concerns/HasFileAttachments.php in the line if ($storage->getVisibility($file) === 'private') why do they ask if it is private if a variable that defines the visibility, then if I do not have access to the acl permissions of that route or I can validate this and it explodes for no reason, this only happens with this component because the upload files does work

**Comment by zxhub-llc** [comment]
Hello, have you checked the permissions and cors on the s3 bucket?
